Documentation/contributing/testing/e2e.rst

```diff
@@ -211,3 +211,14 @@ has been resolved):
     # make modules_install
 
 Finally, you can use the instructions from :ref:`the previous chapter<test_cilium_on_lvh>` to run and to test Cilium.
+
+Cleaning up tests
+^^^^^^^^^^^^^^^^^
+
+For cleaning up, you can simply delete the namespace for connectivity tests.
```

Comment:
My understanding is that if the tests complete then the namespace will be deleted automatically, so this cleanup is necessary only if you interrupt tests? If that's correct, maybe it's worth mentioning it here.

Reply:
agree, addressed in https://github.com/cilium/cilium/pull/30428/commits/05a8e9c10641b39a15be8bb20a1aad95873d7a4f